src/components/shared/Profile/index.jsx

```diff
@@ -0,0 +1,51 @@
+import { useDeleteLogOutMutation } from "services/targetApi";
+import { revokeCredentials } from "store/auth.reducer";
+import { useDispatch } from "react-redux";
+import Title from "components/shared/Title";
+import styles from "./index.module.scss";
+import profileIcon from "../../../assets/icons/profileIcon.svg";
+import profileBackground from "../../../assets/icons/profileBackground.svg";
+import Button from "components/shared/Button";
+import SplitBar from "components/shared/SplitBar";
+import Description from "components/shared/Description";
+
+const Profile = () => {
+  const dispatch = useDispatch();
+
+  const [logOutRequest] = useDeleteLogOutMutation();
+
+  async function logOut() {
+    try {
```

Comment:
you can use redux matchers to avoid using try/catch here.

Reply:
Will investigate more about redux matchers to implement it on the next PR. Thanks for the suggestion!